package com.bankManagement.customer_service.exception;

public class BankingException extends RuntimeException {
    public BankingException(String message) {
        super(message);
    }
}
